### PR TITLE
added factory girl, pry, and setup rspec test skeleton

### PR DESCRIPTION
**Pull request by ieric01**
-Added factory girl to generate instances of a class 
-Added pry to debug
-Created rspec test skeleton for pieces is_obstructed method

: )

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -41,6 +41,8 @@ gem 'codeclimate-test-reporter', group: :test, require: nil
 
 group :development, :test do
   gem 'rspec-rails', '~> 3.0'
+  gem 'factory_girl_rails'
+  gem  'pry-rails'
```
Comment (mscoutermarsh) [inline Gemfile]: looks like 1 extra space here :) before 'pry-rails'

Comment (mscoutermarsh) [inline Gemfile]: btw this is the kind of stuff Rubocop will catch once we have it

File: app/models/piece.rb [hunk 2]
```diff
@@ -2,5 +2,13 @@ class Piece < ActiveRecord::Base
...
+	enum status: [:black, :white]
+
+  def is_obstructed(destination_x, destination_y)
+    # binding.pry
+  end
+
 end
+
+# p = Piece.new
+# p.is_obstructed('f', 3)
```
Comment (mscoutermarsh) [inline app/models/piece.rb]: could you delete the commented out code :smile:

**Comment by mscoutermarsh** [comment]
Love it, good stuff :). I left a couple style comments. Merge it whenever you're ready :)

**Comment by ieric01** [comment]
Great, I just updated the tests and it now looks  sleek

**Pull request merged**